bytes/bytes/config.py

```diff
@@ -31,12 +31,14 @@ class Settings(BaseSettings):
     rfc3161_provider: Optional[str]
     rfc3161_cert_file: Optional[Path]
 
-    encryption_middleware: EncryptionMiddleware = EncryptionMiddleware.NACL_SEALBOX
-    kat_private_key_b64: str = ""
-    vws_public_key_b64: str = ""
+    encryption_middleware: EncryptionMiddleware = (
+        EncryptionMiddleware.NACL_SEALBOX
+    )  # Should we default to no encryption? Does NACL_SEALBOX require keys?
```

Comment:
Doing encryption requires keys, so not doing encryption as the default makes more sense because that works out of the box.